src/pykiso/lib/connectors/cc_pcan_can.py

```diff
@@ -80,6 +83,9 @@ def __init__(
         :param interface: python-can interface modules used
         :param channel: the can interface name
         :param state: BusState of the channel
+        :param trace_path: path to write the trace
+        :param trace_size: maximum size of the trace in (Mb)
+        :param segmented: Traced data is distributed in several files with size trace_size
```

Comment:
I was a bit confused regarding this option, because I do not see any point to use single trace, if you can use segmented option with larger trace size. I just assumed that it was more flexible this way and as it does not request more ressource, may be it's better to let it like this ? 
However, I'l still add the check of trace size :)

Reply:
Maybe I don't understand the option .

What happens If it don't set segmented and  trace_size is 1 MB but my Logfile has data for 10 MB.

Would it then just create one file with 1 MB and I loose the rest of the log ?  If you keep it like this then mention that information may get loose.

Reply:
It will just save the first MB of information then once limit has been reached, nothing will be saved anymore :)